src/electron/views/device-connect-view/components/device-connect-view-container.tsx

```diff
@@ -26,34 +31,48 @@ export type DeviceConnectViewContainerProps = {
     userConfigurationStoreData: UserConfigurationStoreData;
     deviceStoreData: DeviceStoreData;
     windowStateStoreData: WindowStateStoreData;
+    featureFlagStoreData: FeatureFlagStoreData;
+    androidSetupStoreData: AndroidSetupStoreData;
 };
 
 export const DeviceConnectViewContainer = NamedFC<DeviceConnectViewContainerProps>(
     'DeviceConnectViewContainer',
     props => {
         return (
-            <div className={deviceConnectView}>
-                <WindowTitle
-                    pageTitle={'Connect to your Android device'}
-                    deps={props.deps}
-                    windowStateStoreData={props.windowStateStoreData}
-                >
-                    <HeaderIcon invertColors deps={props.deps} />
-                </WindowTitle>
-                <div className={mainContentWrapper}>
-                    <DeviceConnectBody
-                        deps={props.deps}
-                        viewState={{
-                            deviceConnectState: props.deviceStoreData.deviceConnectState,
-                            connectedDevice: props.deviceStoreData.connectedDevice,
-                        }}
-                    />
-                    <TelemetryPermissionDialog
-                        deps={props.deps}
-                        isFirstTime={props.userConfigurationStoreData.isFirstTime}
-                    />
-                </div>
-            </div>
+            <FlaggedComponent
+                featureFlagStoreData={props.featureFlagStoreData}
+                deps={props.deps}
+                featureFlag={UnifiedFeatureFlags.adbSetupView}
+                enableJSXElement={<AndroidSetupStepComponent {...props} />}
+                disableJSXElement={productionDeviceConnectView(props)}
+            ></FlaggedComponent>
         );
     },
 );
+
+const productionDeviceConnectView = (props: DeviceConnectViewContainerProps) => {
+    return (
+        <div className={deviceConnectView}>
+            <WindowTitle
```

Comment:
The WindowTitle will be shared, although the page title might be different(?). Need to decide what's best

Reply:
When you say it might be different, this means it would change for the different steps of the setup? Or do you mean something else?

Reply:
At first I thought it might change for the different steps of the setup, but that doesn't seem right anymore. So we will probably be able to leave the WindowTitle out of the FlaggedComponent as Dan suggests